release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -639,6 +639,13 @@ For more information, see xref:../edge_computing/ztp-deploying-far-edge-sites.ad
 [id="ocp-4-16-insights-operator_{context}"]
 === Insights Operator
 
+[id="ocp-4-16-etcd-certificates_{context}"]
+=== Security
+
+A new signer certificate authority (CA), `openshift-etcd` is now available to sign certificates. It is contained in a trust bundle with the existing CA. Two CA secrets,`etcd-signer` and `etcd-metric-signer`, are also available for rotation.
+
+Starting with this release, all certificates will move to a proven library. This change allows for the automatic rotation of all certificates that were not managed by `cluster-etcd-operator`. All node-based certificates will continue with the current update process.
```

Comment:
Not familiar with how release notes work, but just wanted to check. If these are two separate things, should they have bullet points or some formatting to indicate that they're not related?
```suggestion
* A new signer certificate authority (CA), `openshift-etcd` is now available to sign certificates. It is contained in a trust bundle with the existing CA. Two CA secrets,`etcd-signer` and `etcd-metric-signer`, are also available for rotation.

* Starting with this release, all certificates will move to a proven library. This change allows for the automatic rotation of all certificates that were not managed by `cluster-etcd-operator`. All node-based certificates will continue with the current update process.
```

Reply:
Hi @skopacz1 I combined this into a single paragraph because they are related...thanks for the input!